Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -1026,18 +1050,40 @@ - (void)scrollToDefaultPosition
 
     NSIndexPath *_Nullable indexPath = [self indexPathOfUnreadMessagesIndicator];
     if (indexPath) {
-        [self.collectionView scrollToItemAtIndexPath:indexPath
-                                    atScrollPosition:UICollectionViewScrollPositionTop
-                                            animated:NO];
+        if (indexPath.section == 0 && indexPath.row == 0) {
```

Comment:
Yup, that's what we use when we enter the view (go to unread indicator if any, otherwise try to scroll to bottom).